mythril/laser/ethereum/strategy/basic.py

```diff
@@ -1,54 +1,80 @@
 """
 This module implements basic symbolic execution search strategies
 """
+from abc import ABC, abstractmethod
+from random import randrange
 
+try:
+    from random import choices
+except ImportError:
+
+    from itertools import accumulate
+    from random import random
+    from bisect import bisect
+
+    def choices(population, weights=None):
```

Comment:
pydoc?

Reply:
Sorry, I didn't get what exactly to be done, if it is comments about the function then I have just added it now.